src/renderer/components/SettingsModal/index.jsx

```diff
@@ -1,98 +1,68 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import Accordion from 'react-bootstrap/Accordion';
 import Col from 'react-bootstrap/Col';
 import Row from 'react-bootstrap/Row';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import Modal from 'react-bootstrap/Modal';
-import { MdSettings } from 'react-icons/md';
+import {
+  MdSettings,
+  MdClose,
+} from 'react-icons/md';
+import { BsChevronExpand } from 'react-icons/bs';
 
 import { getDefaultSettings } from './SettingsStorage';
 
-/** Validate that n_workers is an acceptable value for Taskgraph.
- *
- * @param  {string} value - value for Taskgraph n_workers parameter.
- * @returns {boolean} - true if a valid value or false otherwise.
- *
- */
-function validateNWorkers(value) {
-  const nInt = parseInt(value);
-  return Number.isInteger(nInt) && nInt >= -1;
-}
-
 /** Render a dialog with a form for configuring global invest settings */
 export default class SettingsModal extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       show: false,
-      localSettings: {
-        nWorkers: '',
-        loggingLevel: '',
-        sampleDataDir: null,
-      },
```

Comment:
I like that we're no longer having a separate copy of the settings in this component state!